packages/firebase_core/ios/firebase_core.podspec

```diff
@@ -18,4 +18,11 @@ A new flutter plugin project.
   s.dependency 'Firebase/Core'
   s.ios.deployment_target = '8.0'
   s.static_framework = true
+
+  s.prepare_command = <<-CMD
```

Comment:
CocoaPods already provide the versions via an `NSBundle` for each podspec, as shown in my other PR - not sure if code gen is needed if we already have the version at runtime